plugins/splunk-on-call/README.md

```diff
@@ -0,0 +1,92 @@
+# Splunk On-Call
+
+## Overview
+
+This plugin displays Splunk On-Call information about an entity.
```

Comment:
```suggestion
This plugin displays Splunk On-Call, formerly VictorOps, information about an entity.
```

Just suggest referring to this as it might help trigger searches if people look for it using the old name.